### PR TITLE
bpo-30757: Improve "How to make stand-alone binaries" FAQ

### PR DESCRIPTION
**Pull request by erlend-aasland**
Based on PR (GH-1158) by Denis Akhiyarov.

Co-authored-by: Brian Curtin <brian@python.org>
Co-authored-by: Berker Peksag <berker.peksag@gmail.com>
Co-authored-by: Erlend E. Aasland <erlend.aasland@innova.no>

https://bugs.python.org/issue30757

### CODE REVIEW

**Comment by erlend-aasland** [comment]
(@pablogsal: I've applied _skip news_ and backport through 3.9 for this, as it is a trivial doc update.)

File: Doc/faq/programming.rst [hunk 1]
```diff
@@ -89,13 +91,13 @@ only contains those built-in modules which are actually used in the program.  It
...
+executables:
 
-* `py2exe <http://www.py2exe.org/>`_ for Windows binaries
-* `py2app <https://github.com/ronaldoussoren/py2app>`_ for Mac OS X binaries
-* `cx_Freeze <https://cx-freeze.readthedocs.io/en/latest/>`_ for cross-platform
-  binaries
+* `cx_Freeze <https://marcelotduarte.github.io/cx_Freeze/>`_ (Cross-platform)
+* `PyInstaller <http://www.pyinstaller.org/>`_ (Cross-platform)
+* `py2exe <http://www.py2exe.org/>`_ (Windows only)
+* `py2app <https://github.com/ronaldoussoren/py2app>`_ (macOS only)
```
Comment (Fidget-Spinner) [inline Doc/faq/programming.rst]: I suggest adding two more:
```suggestion
* `Nuitka <https://nuitka.net/>`_ (Cross-platform)
* `PyOxidizer <https://pyoxidizer.readthedocs.io/en/stable/>`_ (Cross-platform)
```

Comment (erlend-aasland) [reply Doc/faq/programming.rst]: Nice, I was unaware of these two; thanks 🙂

**Review by Fidget-Spinner — APPROVED** [review]
LGTM. Thank you!

**Comment by erlend-aasland** [comment]
> LGTM. Thank you!

As always, thank you for your excellent reviews! 🙏🏻

**Review by glaukiol1 — APPROVED** [review]

**Review by pablogsal — APPROVED** [review]
Great job!

**Pull request merged**